src/psyclone/dynamo0p3.py

```diff
@@ -459,17 +463,26 @@ def __init__(self, arg_type):
                                   arg_type.args[2].name, arg_type))
             self._function_space1 = arg_type.args[2].name
 
-            # The optional 4th argument is a stencil specification
+            # The optional 4th argument is either a stencil specification
+            # or a mesh identifier (for inter-grid kernels)
             if len(arg_type.args) == 4:
                 try:
-                    stencil = self._get_stencil(arg_type.args[3],
-                                                VALID_STENCIL_TYPES)
+                    from psyclone.parse import get_stencil, get_mesh
```

Comment:
These changes are part of an __init__ routine which does not specify its arguments for phinx in a docstring. As this was implemented at about the same time as the rule change on documenting interfaces and as I'm actually not sure if we are meant to document __init__ routines, I will let this pass :-).